packages/main/src/ColorPalette.ts

```diff
@@ -43,6 +43,7 @@ import ColorPaletteDialogCss from "./generated/themes/ColorPaletteDialog.css.js"
 interface IColorPaletteItem extends HTMLElement, ITabbable {
```

Comment:
seems like this interface is redundant for the moment and we can move only ITabbable interface implementation to the ColorPaletteItem

Reply:
as discussed offline, will be addressed in other change